Solutions/IronNet IronDefense/Workbooks/IronDefenseAlertDashboard.json

```diff
@@ -0,0 +1,1114 @@
+                },
+                {
+                  "durationMs": 7776000000
+                }
+              ]
+            },
+            "value": {
+              "durationMs": 7776000000
+            }
+          },
+          {
+            "id": "86b16bbf-4f81-47c9-8520-f74207a6be96",
+            "version": "KqlParameterItem/1.0",
+            "name": "Category",
+            "type": 1,
+            "isHiddenWhenLocked": true,
+            "value": ""
+          },
+          {
+            "id": "0814f935-9f4c-4db4-aada-dd01c2051412",
+            "version": "KqlParameterItem/1.0",
+            "name": "Subcategory",
+            "type": 1,
+            "isHiddenWhenLocked": true,
+            "value": ""
+          },
+          {
+            "id": "44cfba2e-f5f8-40e9-b890-21d01a2e6dad",
+            "version": "KqlParameterItem/1.0",
+            "name": "IronVueUrl",
+            "type": 1,
+            "isHiddenWhenLocked": true,
+            "value": ""
+          },
+          {
+            "version": "KqlParameterItem/1.0",
+            "name": "Time",
+            "type": 1,
+            "isHiddenWhenLocked": true,
+            "timeContext": {
+              "durationMs": 86400000
+            },
+            "id": "5265fb7c-9ecd-454b-93d9-41a87e3c6eba",
+            "value": ""
+          },
+          {
+            "id": "91d5556f-c021-4d14-b603-87607af0964a",
+            "version": "KqlParameterItem/1.0",
+            "name": "pr10",
+            "type": 1,
+            "isHiddenWhenLocked": true,
+            "criteriaData": [
+              {
+                "criteriaContext": {
+                  "operator": "Default",
+                  "rightValType": "param",
+                  "resultValType": "expression",
+                  "resultVal": "{token} > 0"
+                }
+              }
+            ]
+          },
+          {
+            "id": "e991e752-07b3-4971-ae88-4301c651acd1",
+            "version": "KqlParameterItem/1.0",
+            "name": "SourceIP",
+            "type": 1,
+            "isHiddenWhenLocked": true,
+            "value": ""
+          }
+        ],
+        "style": "pills",
+        "queryType": 8
+      },
+      "name": "parameters - 3"
+    },
+    {
+      "type": 1,
+      "content": {
+        "json": "<h1>Alert ID : {AlertId}</h1>"
+      },
+      "name": "text - 5"
+    },
+    {
+      "type": 11,
+      "content": {
+        "version": "LinkItem/1.0",
+        "style": "tabs",
+        "links": [
+          {
+            "id": "217b6954-972d-4acc-9f19-6fbbc1908379",
+            "cellValue": "tabchange",
+            "linkTarget": "parameter",
+            "linkLabel": "Statistics",
+            "subTarget": "statistics",
+            "style": "link"
+          },
+          {
+            "id": "b2eb7d04-e625-45d7-bbcb-20b38e276edb",
+            "cellValue": "tabchange",
+            "linkTarget": "parameter",
+            "linkLabel": "Details",
```

Comment:
I am getting invalid URL message when clicking on Details tab. Is this the expected behavior? Please confirm.

![image](https://user-images.githubusercontent.com/73389082/125843001-ffdd643d-593c-49bd-9db3-34141022efc4.png)

Reply:
This is expected the configuration steps will be mentioned in the user manual to form a valid URL